packages/zen-bull-netting/test/unit-test/ZenBullNettingUnit.t.sol

```diff
@@ -49,16 +58,14 @@ contract ZenBullNettingUnit is ZenBullNettingBaseSetup {
     }
 
     function testSetMinWethAmount() public {
-        vm.prank(owner);
-        zenBullNetting.setMinWethAmount(5e18);
-
-        assertEq(zenBullNetting.minWethAmount(), 5e18);
+        _setMinWethAmount();
+        assertEq(zenBullNetting.minWethAmount(), minWeth);
```

Comment:
Curious why setting this as state but not minBullAmount etc

Reply:
@alpinechicken why you mean? just the decision of why to create a function/have a variable have minWethAmount?

Reply:
Ah u mean `minWeth` variable, just because I needed to add before some tests to make sure the min amount is set, will add also `minBull` later in the Bull queueing PR, and maybe later move all of this inside the setup function...